app/pages/components/Layout.tsx

```diff
@@ -43,8 +44,7 @@ const Layout: FC = (props) => {
     <ConnectionProvider endpoint={endpoint}>
       <WalletProvider wallets={wallets} autoConnect>
         <WalletModalProvider>
-          <Navbar />
-
+          <Landing />
           <main>{children}</main>
```

Comment:
So this `Layout` file is a Next.js concept where you would put content/logic that gets rendered on every page of your app. The routing is handled by Next.js and it looks at the files in the `pages` directory to auto-magically create your routes. 

So you're going to want to move the `Landing` content to `pages/index.tsx` because that file serves the root route (`/`). It'll go right here: https://github.com/MikaelCarpenter/NFT-Club/blob/main/app/pages/index.tsx#L69. And because this Landing content isn't reused anywhere it doesn't need to be a component, you can just move the JSX right into `index.tsx`

Additional routing docs in case the way I described it is unclear: https://nextjs.org/docs/routing/introduction

Reply:
This was super helpful, thanks!